### ISSUE TITLE
Streamline logging across all Arcus scripts

### ISSUE THREAD

**Issue by stijnmoreels** [arcus-azure/arcus.scripting#184]
**Is your feature request related to a problem? Please describe.**
Is it an idea to add an `[Arcus] ` prefix on all our logging, so people now it comes from us?
When the script is used in a jungle of other stuff, it maybe useful to have some 'scoped' loggings, which we could achieve with this.

Thoughts?

**Comment by pim-simons** [comment]
You mean instead of doing `Write-Host "Attempting to enable $LogicAppName"` you would do `[Arcus] Write-Host "Attempting to enable $LogicAppName"`?

For example, the output from Remove-AzApiManagementDefaults script will look like this:
```
[Arcus] Start removing Azure API Management defaults...
[Arcus] Checking if 'echo' API exists...
[Arcus] The 'echo' API does not exist, skipping removal...
[Arcus] Checking if 'starter' product exists...
[Arcus] The 'starter' product does not exist, skipping removal...
[Arcus] Checking if 'unlimited' product exists...
[Arcus] The 'unlimited' product does not exist, skipping removal...
[Arcus] Finished removing Azure API Management defaults!
```

It seems like a reasonable enhancement, especially when used in other stuff. 
@stijnmoreels I can create a PR for this so we can also see the results in our integration and unit tests and see if this is something we consider an improvement 😉?

**Comment by stijnmoreels** [comment]
Yeah, seems like this is probably more a discussion topic 😅 . Maybe we should ask some other users/contributors if this would indeed be an enhancement.

**Comment by stijnmoreels** [comment]
But in short, yes, the example you shown was my proposal. Definitely when they use different types of PowerShell stuff, it can come complex, maybe. But not sure if this is required for many.
Also, not a very critical issue, IMO.

**Comment by pim-simons** [comment]
> Yeah, seems like this is probably more a discussion topic 😅 . Maybe we should ask some other users/contributors if this would indeed be an enhancement.

If its more of a discussion, should we close this and move it to https://github.com/arcus-azure/arcus.scripting/discussions? Seems like a good fit 😁
